src/OID.cpp

```diff
@@ -463,12 +478,13 @@ int main(int argc, char* argv[]) {
   std::string configGenOption;
   std::optional<fs::path> jsonPath{std::nullopt};
 
+  std::map<Feature, bool> features = {
+      {Feature::PackStructs, true},
```

Comment:
I didn't really get to the bottom of what this does (and why we'd ever want to disable it), but I left it there for now for consistency.